src/elements/TableLinkCell/TableLinkCell.js

```diff
@@ -50,7 +50,7 @@ const TableLinkCell = ({
   const { value: stateValue, label: stateLabel, className: stateClassName } = item.state ?? {}
 
   return (
-    <td className={tableCellClassNames}>
+    <td data-testid={data.headerId || data.type} className={tableCellClassNames}>
```

Comment:
It's not correct to set the `type` as an ID, because we might have 2 columns with the same type

Reply:
@mavdryk  type is removed. added headerId.